mage/query-modules/cpp/refactor.md

```diff
@@ -0,0 +1,595 @@
+MERGE (p2:Person{name:'person2'}) 
+MERGE (p3:Person{name:'person3'}) 
+MERGE (p4:Person{name:'person4'}) 
+MERGE (p5:Person{name:'person5'}) 
+MERGE (p6:Person{name:'person6'}) 
+CREATE (ana)-[:KNOWS]->(p2)-[:KNOWS]->(p3)-[:KNOWS]->(p4) 
+CREATE (p4)<-[:KNOWS]-(p5) 
+CREATE (marija)-[:KNOWS]->(p6);
+```
+
+</TabItem>
+
+<TabItem value="input">
+
+The image below shows the above data as a graph:
+
+<img src={require('../../data/query-modules/cpp/refactor/clonesubgraph1.png').default}/>
+    
+</TabItem>
+
+<TabItem value="run">
+
+```cypher
+MATCH (ana:Ana),
+      (p2:Person{name: "person2"}),     
+      (p3:Person{name: "person3"}),
+      (p4:Person{name: "person4"}),
+      (p5:Person{name: "person5"})
+CALL refactor.clone_subgraph([ana, p2, p3, p4, p5], [], {
+       standinNodes:[[ana, marija]]
+   })
+YIELD * RETURN *;
+```
+
+</TabItem>
+
+<TabItem value="result">
+
+The results should be identical to the ones on graph below, except for the
+`id` values that depend on the internal database `id` values. Note that the 
+whole subgraph was cloned except for node `:Ana` because node `:Marija` was 
+used as its "stand in" node.
+
+<img src={require('../../data/query-modules/cpp/refactor/clonesubgraph2.png').default}/>
+
+</TabItem>
+
+</Tabs>
+
+
+## Example - refactor.clone_subgraph_from_path
+
+<Tabs
+  groupId="clone_subgraph_path_example"
+  defaultValue="input"
+  values={[
+    {label: 'Step 1: Cypher load commands', value: 'load'},
+    {label: 'Step 2: Input graph', value: 'input'},
+    {label: 'Step 3: Running command', value: 'run'},
+    {label: 'Step 4: Results', value: 'result'},
+  ]
+}>
+
+<TabItem value="load">
+
+You can create a simple graph database by running the following queries:
+
+```cypher
+MERGE (ana:Ana{name:'Ana'}) 
+MERGE (marija:Marija{name:'Marija'}) 
+MERGE (p2:Person{name:'person2'}) 
+MERGE (p3:Person{name:'person3'}) 
+MERGE (p4:Person{name:'person4'}) 
+MERGE (p5:Person{name:'person5'}) 
+MERGE (p6:Person{name:'person6'}) 
+CREATE (ana)-[:KNOWS]->(p2)-[:KNOWS]->(p3)-[:KNOWS]->(p4) 
+CREATE (p4)<-[:KNOWS]-(p5) CREATE (p5)-[:LOVES]->(p6) 
+CREATE (marija)-[:KNOWS]->(p6);
+```
+
+</TabItem>
+
+<TabItem value="input">
+
+The image below shows the above data as a graph:
+
+<img src={require('../../data/query-modules/cpp/refactor/clonesubgraphpath1.png').default}/>
+    
+</TabItem>
+
+<TabItem value="run">
+
+```cypher
+MATCH (ana:Ana),
+      (marija:Marija)
+MATCH path = (ana)-[:KNOWS*]->(node)
+WITH ana, marija, collect(path) as paths
+CALL refactor.clone_subgraph_from_paths(paths, {
+       standinNodes:[[ana, marija]]
+})
+YIELD cloned_node_id, new_node
+RETURN cloned_node_id, new_node;
+```
+
+</TabItem>
+
+<TabItem value="result">
+
+The results should be identical to the ones on graph below, except for the
+`id` values that depend on the internal database `id` values. Note that the 
+whole subgraph was cloned except for node `:Ana` because node `:Marija` was 
+used as its "stand in" node.
```

Comment:
```suggestion
The results should be identical to the ones in the graph below, except for the
`id` values, which depend on the internal database `id` values. Note that the 
whole subgraph was cloned except for the node `:Ana` because node `:Marija` was 
used as its "stand-in" node.
```